packages/twilio-run/__tests__/config/utils/mergeFlagsAndConfig.test.ts

```diff
@@ -3,7 +3,7 @@ import '../../../src/config/utils/mergeFlagsAndConfig';
 import { mergeFlagsAndConfig } from '../../../src/config/utils/mergeFlagsAndConfig';
 
 const baseFlags = {
-  config: '.twilio-functions',
+  config: '.twilioserverlessrc.js',
```

Comment:
Everywhere else, except in this test, the config is referred to as `.twilioserverlessrc` without the `.js`. Is that an issue?

Reply:
No. The parser should support `.js`, `.json`, `.json5`, etc. I can change to for consistency though